### ISSUE TITLE
feature request: cardio info

### ISSUE THREAD

**Issue by nutpantz** [brodeurlv/fastnfitness#70]
I would love to see some more cardio options found on cardio machines.
Info like,  program or workout' # or name, calories, max heartrate, min heartrate, resistance level. Etc.

**Comment by brodeurlv** [comment]
Hi @nutpantz , 

"Resistance level" is something that would make sense and that is on almost all cardio machines. The other item you are listing would be more on a free note part I guess. 

PS: Please start your messages with "Hello" or "Hi" to increase your chance to have a feedback. Thanks.

**Comment by Forbo** [comment]
I agree, the ability to track resistance for cardio workouts would be an awesome addition!
